python/test/unit/language/test_core.py

```diff
@@ -1477,13 +1487,23 @@ def kernel(Z, X, OLD, AXIS: tl.constexpr, SHAPE0: tl.constexpr, SHAPE1: tl.const
     z = numpy_random(z_shape, dtype_str=dtype_x_str, rs=rs)
     old = np.zeros(z_shape, dtype=getattr(np, dtype_x_str))
     # reference results
-    z_ref = z + np.sum(x, axis=axis, keepdims=False)
+    if x.dtype == np.float16:
```

Comment:
We now need to add `float16` in `dtype_x_str` in L1456?

Reply:
you're right; I guess I somehow removed it while I was tidying up the PR. Added it back and verified that the tests still pass!